src/components/black-friday/PlatformSection.tsx

```diff
@@ -1,49 +1,75 @@
 import React from 'react';
 import styles from '@/components/black-friday/BF-HeroSection.module.scss';
+import { getImage } from '@/lib/getImage';
+import Image from 'next/image';
+
 
 const PlatformSection = ({ textContent }) => {
+  const platforms = [
+    {
+    title: textContent.PlatformSection.web,
+    image: getImage('/images/special-offer/black-friday/Globe.svg'),
+    alt: 'Web image'
+    },
+    {
+    title: textContent.PlatformSection.iOS,
+    image: getImage('/images/special-offer/black-friday/iOS.svg'),
+    alt: 'iOS icon'
+    },
+    {
+    title: textContent.PlatformSection.android,
+    image: getImage('/images/special-offer/black-friday/Android.svg'),
+    alt: 'Android image'
+    },
+    {
+    title: textContent.PlatformSection.mac,
+    image: getImage('/images/special-offer/black-friday/MacOs.svg'),
+    alt: 'iOS icon'
+    },
+    {
+    title: textContent.PlatformSection.windows,
+    image: getImage('/images/special-offer/black-friday/Windows.svg'),
+    alt: 'Windows image'
+    },
+    {
+    title: textContent.PlatformSection.linux,
+    image: getImage('/images/special-offer/black-friday/Linux.svg'),
+    alt: 'Linux image'
+    }
+  ];
+
+
   return (
-    <section className="overflow-hidden">
+    <section className="overflow-hidden bg-gray-1">
+      
       <div className="flex flex-col items-center py-12">
         <div className="center flex items-center px-20 pb-16 text-center">
-          <p className="text-4xl font-semibold text-white">{textContent.PlatformSection.title}</p>
+          <p className="text-5xl font-semibold text-gray-100">
+            {textContent.PlatformSection.title}
+          </p>
+        </div>
+        <div className="center flex items-center px-20 pb-16 text-center">
```

Comment:
I don't think `center` exists in tailwindCSS. Should be `text-center`, `bg-center`, `justify-center`, etc